### DISCUSSION TITLE
Automatically mark as read

### DISCUSSION THREAD

**Discussion by johannes1984** [FreshRSS/FreshRSS#5550] [Q&A]
Hi,
I'm wondering if there is any function to automatically mark articles as read after a certain time?

Example:
Sometimes I do not have the time to read through everything and then things get outdated. So it would be great to have something run in the background and marking everything as read which is older than x hours or x days. Similar to the "Older than one day" under the the "Mark as read" button.

Any ideas? :-)

Johannes

**Comment by Alkarex** [comment]
Hello,
There are several options to automatically mark as read, including keeping a maximum number of unread articles, or when it is no longer in the upstream news feed:
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/80730418-b38c-41ce-a0b8-9d14452e23ca)
